ocs_ci/ocs/resources/pod.py

```diff
@@ -2218,6 +2220,20 @@ def wait_for_storage_pods(timeout=200):
         and "storageclient" not in pod.name
     ]
 
+    # Filter out job pods if requested
+    if skip_job_pods:
+
+        def is_job_owned(pod_obj):
+            """Check if pod is owned by Job controller"""
+            try:
+                pod_data = pod_obj.get()
+                owner_refs = pod_data.get("metadata", {}).get("ownerReferences", [])
+                return any(ref.get("kind") == "Job" for ref in owner_refs)
+            except Exception:
+                return False
```

Comment:
Can you export it as a helper function here in the `pod` file or the `helper` file?

Reply:
why? it is very specific to pods?